src/run_throttlebot.py

```diff
@@ -959,7 +961,7 @@ def parse_resource_config_file(resource_config_csv, sys_config):
                 # Convert REPR to RAW AMOUNT
                 if amount_repr == 'PERCENT':
                     if amount <= 0 or amount > 100:
-                        print 'Error: invalid default percentage. Exiting...'
+                        logging.warning('Error: invalid default percentage. Exiting...')
```

Comment:
This should be at the error level, it causes the program to exit

"Error" is redundant